projects/go-lib/styles/_typography.scss

```diff
@@ -11,6 +11,7 @@ strong {
 .go-heading {
   font-family: $primary-font-stack;
   font-weight: $weight-regular;
+  line-height: 1.4;
```

Comment:
So I looked through the Goponent's repo in an effort to be consistent and it looks like we do `1.3` in most places for line-height. (honestly we should probably have a variable for that). What do you think about changing this to `1.3`?

Additionally, this will affect the height of the heading elements through all the apps. What do you think about modifying the `$heading-margin-bottom` variable to say `1rem` to help accommodate for the increase in size? It will still be slightly larger, but I think that's fine and honestly looks better than the `1.5rem` it's at now.

Reply:
I like that idea